### ISSUE TITLE
Add hint as to whether an accidental has been deliberately displayed or hidden

### ISSUE THREAD

**Issue by mdgood** [w3c/musicxml#108]
MusicXML represents whether accidentals appear or not, but nothing indicates if this is a result of applying standard music notation rules, or a deliberate choice to display or hide an accidental. This hint could be helpful for higher-quality exchange between interactive applications doing transposition and other types of formatting. 

This will run into similar problems as a stem direction hint in terms of there being no uniform cross-application definition of a "usual direction". However, like that hint, it can suggest why an engraving is done a certain way, in a form that is both relatively simple and reasonably portable across applications.

This issue was raised in multiple conversations with MusicXML developers at the NAMM and Musikmesse shows, as well as via private email.

**Comment by joeberkovitz** [comment]
I agree that this is a problem and Noteflight's data model has had to address this concern separately, as a further validation that the issue is significant. However, I feel that this falls into a large category of semantic/visual gray areas in MusicXML that need more careful thought before being resolved.

I do not see this issue as a showstopper for interop. MusicXML has lacked this capability for some time and it does not prevent import/export from working acceptably, nor has it encouraged developers to roll their own out-of-band mechanism. Until authors modify the measure containing the accidental, the issue is not detectable, and when they do, they are in a position to correct it in the host application.

**Comment by artexanis** [comment]
"but nothing indicates if this is a result of applying standard music notation rules, or a deliberate choice to display or hide an accidental"

Which rules? There are a lot of (conflicting) rules. (eg: http://lilypond.org/doc/v2.12/Documentation/user/lilypond/Displaying-pitches#Automatic-accidentals)

I think we need this because it's not clear what 'standard music notation rules' are. Lilypond has 14 different ways of placing accidentals.

Maybe we can specify which 'accidental method' was used (in the 'defaults' element). It's easy to spot divergent accidentals and handle them correctly.
